server/executor/trigger/traceid.go

```diff
@@ -0,0 +1,66 @@
+package trigger
+
+import (
+	"context"
+	"fmt"
+	"github.com/kubeshop/tracetest/server/expression"
+	"github.com/kubeshop/tracetest/server/model"
+)
+
+func TRACEID() Triggerer {
+	return &traceidTriggerer{}
+}
+
+type traceidTriggerer struct{}
+
+func (t *traceidTriggerer) Trigger(ctx context.Context, test model.Test, opts *TriggerOptions) (Response, error) {
+	response := Response{
+		Result: model.TriggerResult{
+			Type: t.Type(),
+		},
+	}
+
+	return response, nil
+}
+
+func (t *traceidTriggerer) Type() model.TriggerType {
+	return model.TriggerTypeTRACEID
+}
+
+func (t *traceidTriggerer) Resolve(ctx context.Context, test model.Test, opts *TriggerOptions) (model.Test, error) {
+	traceid := test.ServiceUnderTest.TRACEID
+
+	if traceid == nil {
+		return test, fmt.Errorf("no settings provided for TRACEID triggerer")
+	}
+
+	id, err := opts.Executor.ResolveStatement(WrapInQuotes(traceid.ID, "\""))
+
+	if err != nil {
+		return test, err
+	}
```

Comment:
Sometimes to associate an error handling with the code that caused the error we put the code together:
(It is not a standard on Go as far as I saw, but I saw a bunch of Go programmers doing that to help this association)

```suggestion
	traceid := test.ServiceUnderTest.TRACEID
	if traceid == nil {
		return test, fmt.Errorf("no settings provided for TRACEID triggerer")
	}

	id, err := opts.Executor.ResolveStatement(WrapInQuotes(traceid.ID, "\""))
	if err != nil {
		return test, err
	}
```